src/themes/elife/lib/downloads.ts

```diff
@@ -22,7 +22,17 @@ const deriveUrl = (type: string, id: string, title = ''): string => {
 const buildLinkToFiguresPdf = (url: string): void => {
   after(
     select('[data-is-download-pdf-link]')[0],
-    create('li', null, create('a', { href: url }, 'Figures PDF'))
+    create('li', null, create('a', { href: url }, 'Figures PDF')),
+    create(
+      'li',
+      null,
+      create('a', { href: url }, 'Executable version'),
```

Comment:
We either need to change the function name from `buildLinkToFiguresPdf` or this belongs in a new function. If this is just placeholder for now then just use `#` as the href rather that the figures pdf url.